src/main/java/entities/Recipe.java

```diff
@@ -0,0 +1,23 @@
+package entities;
+
+import java.util.ArrayList;
+
+public class Recipe {
+    private ArrayList<String> steps;
```

Comment:
I think we want getters for these instance variables. We might also want some other setters, like deleteStep (i.e. if I add the wrong step by mistake, can I fix my mistake?)